10up/ElasticPress
# Make failed queries notice dismissible

### Description of the Change
This PR makes the status report notice dismissible. 



Closes #3322

### How to test the Change
1.- Trigger a failed query
2.- Go to the admin dashboard
3.- Dismiss the notice
4.- Reload page. Notice should not re-appear
5.- Trigger another error
6.- Notice should re-appear



### Changelog Entry

> Fixed - Make notice dismissible


### Credits

Props @oscarssanchez, @felipeelia 


### Checklist:
